File: core/common/core/src/androidMain/kotlin/org/michaellang/common/provider/DateTimeProviderImpl.kt
package org.michaellang.common.provider

class DateTimeProviderImpl : DateTimeProvider {
    override fun getTimeMills(): Long {
        return System.currentTimeMillis()
    }

    override fun parse(date: String, format: String): Long {
        return 11111
    }

    override fun getCurrentDateTime(format: String, timeZone: String?): String {
        return "200-2220"
    }

    override fun format(date: Long, dateFormat: String): String {
        return "200-2220"
    }

}

File: platform/ios/covid/covid/Provider/DateTimeProviderImpl.swift
//
//  DateTimeProvider.swift
//  covid
//
//  Created by michael on 17.05.2020.
//  Copyright © 2020 michael. All rights reserved.
//

import Foundation
import KotlinShared

class DateTimeProviderImpl: DateTimeProvider {
    func format(date: Int64, dateFormat: String) -> String {
        return "123"
    }
    
    func getCurrentDateTime(format: String, timeZone: String?) -> String {
        return "123"
    }
    
    func getTimeMills() -> Int64 {
        return 123
    }
    
    func parse(date: String, format: String) -> Int64 {
        return 123
    }
    
}
